northstar-runtime/src/runtime/console/mod.rs

```diff
@@ -405,15 +405,14 @@ where
 
             // The codec might have pulled bytes in the the read buffer of the connection.
             if !stream.read_buffer().is_empty() {
```

Comment:
Should this `if` be the while loop, the comment below talks about?

Reply:
Don't think so because this is all just about dealing with some bytes being in the buffer of the `Framed`. The loop afterwards wraps the (unbuffered) `AsyncRead` and the buffers aren't used.